.github/workflows/validation.yml

```diff
@@ -0,0 +1,104 @@
+name: Repos Validation
+
+on:
+  pull_request_target:
```

Comment:
Note) `pull_request_target` is triggerd if a PR is from a fork repo.
This is required to use `secrets.GITHUB_TOKEN` safely.
https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#pull_request_target